### ISSUE TITLE
net_g Synthesizer generates a horrible noise

### ISSUE THREAD

**Issue by AznamirWoW** [fumiama/Retrieval-based-Voice-Conversion-WebUI#87]
**To Reproduce**

1) train a new voice model using a single 3 sec silent audio file, 1 epoch. Run inference on the same 3 sec silent audio using the model you created. Analyze the inference result using a spectrogram. Notice solid lines at several frequency levels.

![image](https://github.com/user-attachments/assets/dbcb577e-4d3d-490c-94c8-2e2b6df34baa)

2) modify RVC code and comment out the line that loads weights from a voice model, thus keeping Synthesizer in a blank slate.
https://github.com/fumiama/Retrieval-based-Voice-Conversion-WebUI/blob/96e3d8af403ca695ba460c71169f4e4c5c901d72/rvc/synthesizer.py#L23

3) run inference on the same 3 sec audio file,  Analyze the inference result using a spectrogram. Notice solid lines at every 1KHz level.

![image](https://github.com/user-attachments/assets/7ba0bd5a-4cc1-4e1b-91fb-6a28a9ab776a)

**Expected behavior**
One would expect that training a model from scratch does not introduce artifacts that are not present in source audio files.

**Screenshots**
see above

**Comment by fumiama** [comment]
Are you using the web ui? In this case, you can check about whether the program still loads the pretrained model (named like f0Dxx.pth). If so, remove them all and try it again.

**Comment by AznamirWoW** [comment]
> Are you using the web ui? In this case, you can check about whether the program still loads the pretrained model (named like f0Dxx.pth). If so, remove them all and try it again.

In my investigation I tried making two models using a silent file - one model using a default pretrained f0D32k.pth+f0G32k.pth 
![image](https://github.com/user-attachments/assets/e9757eb5-604d-4be4-953b-2f6de445707c)

and one model without using a default pretrain.
![image](https://github.com/user-attachments/assets/0ec0de80-3995-426f-8d54-a663d707c83a)


 Then I tried using inference using the resulting models (no index) and since the result was different I'm sure no weights were used. Also since I've manually disabled a load of weights into synthesizer's state_dict that test's result was also different, and much worse, than the test using trained models.

**Comment by AznamirWoW** [comment]
there's some difference in spectrograms using different upsample_rates and upsample_kernel_sizes, but I could not quite eliminate the overlapping

upsample_rates = [10, 8, 2, 2]

10x 100Hz lines = 1KHz band
8x 1KHz lines = 8KHz band
2x 8KHz lines = 16KHz band
2x = 32KHz final upsample

so there are 10 faint lines within each 1KHz band, they overlap at each KHz, so each KHz there's twice as strong line
and two 8KHz bands overlap at 8KHz, creating 4x strong line

some examples:

10,8,4,4
![image](https://github.com/user-attachments/assets/0e382b4f-2f79-4d4d-b7e9-1421ca12087d)

10,8,4,2
![image](https://github.com/user-attachments/assets/5b078446-f4ca-4adb-826c-70560c8aea4a)

20,16,4,4
![image](https://github.com/user-attachments/assets/94cf7276-a2e0-4a41-988d-c3f1ab370d1f)

**Comment by SayanoAI** [comment]
How are you rendering those graphs in audacity? I was not able to replicate this using the default mel spec in audacity (voice model applied, original, blank net_g):

![image](https://github.com/user-attachments/assets/afdc9fdf-28b3-4571-a8fe-149449bfd3dd)

**Comment by AznamirWoW** [comment]
> How are you rendering those graphs in audacity? I was not able to replicate this using the default mel spec in audacity (voice model applied, original, blank net_g):

I'm using Spek 
https://www.spek.cc/p/download

**Comment by AznamirWoW** [comment]
> How are you rendering those graphs in audacity? I was not able to replicate this using the default mel spec in audacity (voice model applied, original, blank net_g):

Audacity also shows it

![image](https://github.com/user-attachments/assets/f04f269f-7298-477b-908a-a698c0fa8d41)

**Comment by SayanoAI** [comment]
Did you do anything special with the settings? Why does mine look black? I'm using the default spectogram settings:
![image](https://github.com/user-attachments/assets/98d3f154-e7c0-4ec2-a40e-29819a002ee6)

**Comment by AznamirWoW** [comment]
> Did you do anything special with the settings? 

It looks like RVC (I used RVC1006Nvidia.7z build) has an issue with inferring a short mute audio - it does not do anything, so just like you I could not reproduce it with just a mute wav. Maybe it automatically apply the index, so the index masks the noise.

But then I've trained a model from scratch, one epoch, no pretrained, and used it without an index to infer a longer audio and here it is:
![image](https://github.com/user-attachments/assets/c74d5155-1671-4d5f-8467-8ae697f74fdb)

model in question: https://drive.google.com/file/d/142jVVY5194kMaxoa6Teayz9nS49DX2-X/view?usp=sharing
audio: https://drive.google.com/file/d/1SmzHLol-ShTvcd-aK5kRP_Yp0q5gILaO/view?usp=sharing
